src/sagemaker_huggingface_inference_toolkit/handler_service.py

```diff
@@ -105,9 +105,13 @@ def load(self, model_dir):
         elif "config.json" in os.listdir(model_dir):
             task = infer_task_from_model_architecture(f"{model_dir}/config.json")
             hf_pipeline = get_pipeline(task=task, model_dir=model_dir, device=self.device)
+        elif "model_index.json" in os.listdir(model_dir):
```

Comment:
Does `model_index.json` always mean text-to-image?

Reply:
The docstring on this method is stale and should be updated.

Reply:
`model_index.json` is specific to `diffusers` model similar to `config.json` for `transformers` models. Without it you cannot use `diffusers`.

Reply:
I mentioned the stale docstring as well, but that's a minor comment I suppose.